libraries/botbuilder-dialogs-adaptive-teams/tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "allowSyntheticDefaultImports": true,
+    "esModuleInterop": true,
+    "lib": ["es2015", "dom"],
```

Comment:
"dom" isn't necessary and could lead to other bugs during development. Can you take this out?

Reply:
removed